src/stores/options.json

```diff
@@ -76,7 +76,7 @@
         "label": "Any Mobile Activity",
         "key": "Any Firefox Non-desktop Activity",
         "disabledDimensions": ["os", "attributed"],
-        "shortDescription": "The profile has sent a telemetry ping from any of apps considered for the 2019 Mobile Reach KPI (Fenix, Fennec Android, Fennec iOS, Firefox Lite, FirefoxConnect, Focus Android, and Focus iOS) on the day in question."
+        "shortDescription": "The profile has sent a telemetry ping from any of apps considered for the 2019 Mobile Reach KPI (Fenix, Fennec Android, Fennec iOS, Firefox Lite (excluding CN), FirefoxConnect, Focus Android, Focus iOS, and Lockwise) on the day in question."
```

Comment:
This is the most important change. It fixes the list of apps considered for the 2019 mobile KPI.